packages/material-ui-unstyled/src/SliderUnstyled/SliderUnstyled.js

```diff
@@ -11,6 +11,7 @@ import {
   unstable_capitalize as capitalize,
   unstable_useControlled as useControlled,
 } from '@material-ui/utils';
+import sliderClasses from './sliderClasses';
```

Comment:
Do we need a different file to host this file? If we do, should we write it in TypeScript directly?

Reply:
I've extracted it in a separate file as it needs to be used in both components files (Slider & SliderValueLabel). I think it's better to have it separately for easier navigation. Converting it to ts makes sense, let me do that 👍

Reply:
Done.

Reply:
There are some ts errors when converting this to ts file - https://app.circleci.com/pipelines/github/mui-org/material-ui/27322/workflows/7aed2605-b9fb-42e8-81d1-5b6e10ccb916/jobs/197097. Let me leave it as is and address as a follow up PR as it seems like we need to do changes with the build